src/tallies/tally_scoring.cpp

```diff
@@ -1048,13 +1024,17 @@ void score_general_ce(Particle& p, int i_tally, int start_index,
               // rxn.products_ array to be exceeded. Hence, we use the size
               // of this array and not the MAX_DELAYED_GROUPS constant for
               // this loop.
-              for (auto d = 0; d < rxn.products_.size() - 2; ++d) {
+              for (auto d = 1; d < rxn.products_.size(); ++d) {
```

Comment:
Say rxn.products.size() is 5. Originally we'd have d \in {0, 1, 2}, consequently indexing into nu with indices {1, 2, 3}. Now d ranges in {1, 2, 3, 4}, with the same indices into nu. So there is one more element being covered now? Is this intentional?

Reply:
Yes, this is indeed intentional (sorry I forgot to mention this in the original PR description). There are two embedded assumptions in this:
1. The first reaction product is not a delayed neutron (it's always the prompt neutron)
2. There is always a photon product at the end

Both assumptions are pretty safe, but I did run into a problem when simulating a problem that had fake data (no fission photons) and I figured it would be good to at least relax assumption 2.